config/config.go

```diff
@@ -11,6 +11,7 @@ var JwtKey string
 var RelayUrl string
 var RelayAuthKey string
 var InvoiceList = "INVOICELIST"
+var BudgetInvoiceList = "BUDGETINVOICELIST"
```

Comment:
it might be worth mentioning with a comment above InvoiceList that those vars are for the cache key term

otherwise it might be a bit unclear for a reader